scripts/basic-workflow-tests.sh

```diff
@@ -312,7 +317,7 @@ test_exec() {
     #   `ilab model convert` is only implemented for macOS with M-series chips for now
     #test_convert
 
-    test_serve "${DATA_HOME}/instructlab/models/ggml-model-f16.gguf"
+    test_serve "${CACHE_HOME}/instructlab/models/ggml-model-f16.gguf"
```

Comment:
This is the trained output if I'm not mistaken so I believe this should be under user data, but I could be wrong here. Maybe @mbestavros could clarify here.

Reply:
Looks like it should be $CACHE_HOME based on the change here: https://github.com/instructlab/instructlab/pull/1762/files#diff-ec289415187eaed5e0c70aac3bb11bc066ae949fa64b16a1dd786359954d06dfR123

Reply:
The general rule _for now_ is that any model we aren't downloading should be in the `checkpoints` directory, which is in `DATA_HOME`. (aka: `$DATA_HOME/checkpoints/model.gguf`)